### ISSUE TITLE
Remove TSLint (deprecated)

### ISSUE THREAD

**Issue by prescottprue** [nrwl/nx#3034]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.
**YOU MAY DELETE THE PREREQUISITES SECTION.**

- [X] I am running the latest version
- [X] I checked the documentation (nx.dev) and found no answer
- [X] I checked to make sure that this issue has not already been filed
- [X] I'm reporting the issue to the correct repository (not related to React, Angular or any dependency)

## Expected Behavior

As described in [their post about it, tslint is now deprecated](https://medium.com/palantir/tslint-in-2019-1a144c2317a9) - instead `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` should be used with eslint. This seems to already be happening in the main .eslintrc, so `tslint` should no longer be needed

## Current Behavior

Tslint is included when generating a workspace, but shouldn't be needed since @typescript-eslint/parser is included and setup within .eslintrc.

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Run `npx create-nx-workspace@latest repro-workspace`
2. See that tslint is within package file

**Comment by AlexAegis** [comment]
It's blocked by the angular CLI. There are some usecases where ESLint is a regression, for example this: https://github.com/typescript-eslint/typescript-eslint/issues/1856

**Comment by prescottprue** [comment]
Would it be possible to only include that when generating an angular project? I was generating a react project, so wasn't aware

**Comment by alexeagle** [comment]
Hi, interesting thread but you got the wrong Alex 🅰️

**Comment by prescottprue** [comment]
@AlexAegis ^

**Comment by sushruth** [comment]
would be nice to be able to specify default linter for the whole workspace.

**Comment by fbaba-nibtravel** [comment]
Also, I manually migrate our workspace to ESLint, but when generating a new library, still generates with tslint. Is there any way to generate with eslint atm or we need to manually change the angular.json for that?

**Comment by brandonroberts** [comment]
Closing as a duplicate of #1699
